### ISSUE TITLE
Dependency of setup script on Cheetah is not handled, setup fails

### ISSUE THREAD

**Issue by KonstantinShemyak** [KonstantinShemyak/python-javatools#5]
Despite #39 in the upstream, dependencies of the setup script itself (Cheetah) are not automatically installed. `python setup.py test` in an environment without Cheetah fails on import:
```
$ python setup.py test
Traceback (most recent call last):
  File "setup.py", line 33, in <module>
    from extras import pylint_cmd, cheetah_build_py_cmd
  File "/home/konstantin/src/python-javatools/extras/__init__.py", line 6, in <module>
    from .cheetah_cmd import cheetah_build_py_cmd
  File "/home/konstantin/src/python-javatools/extras/cheetah_cmd.py", line 25, in <module>
    from Cheetah.Compiler import Compiler
ImportError: No module named Cheetah.Compiler
```

Workaround: install Cheetah.

**Comment by KonstantinShemyak** [comment]
`install_requires` is ignored when overriding `cmdclass`??
http://stackoverflow.com/questions/21915469/python-setuptools-install-requires-is-ignored-when-overriding-cmdclass#22179371

**Comment by KonstantinShemyak** [comment]
Closed by 6715cce.
